src/bogo_shim/config.json

```diff
@@ -183,6 +192,7 @@
 
         "DTLS-StrayRetransmitFinished-ClientFull": "Needs investigation",
         "DTLS-StrayRetransmitFinished-ServerResume": "Needs investigation",
+        "DTLS-Replay-NonMonotonic": "Needs investigation, started failing after https://github.com/google/boringssl/commit/f94f3ed3965ea033001fb9ae006084eee408b861",
```

Comment:
This should be investigated!

Reply:
I think this is ok - we just need to have a DTLS window that's at least as big as what the test expects. I can take this. #4079